Userland/Libraries/LibWeb/Layout/BlockFormattingContext.cpp

```diff
@@ -351,6 +351,40 @@ void BlockFormattingContext::compute_height(Box& box)
     box.set_height(height);
 }
 
+void BlockFormattingContext::compute_position(Box& box)
+{
+    // 9.4.3 Relative positioning
```

Comment:
We're not checking that `box.computed_values().position() == CSS::Position::Relative`. This sounds like it should only apply in those cases(?)

Reply:
Ah that's right - I misread the text in 10.3 that refers to 9.4.3, and thought it applied to more contexts. Will fix.